# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
float32 x_current	# in meters
float32 y_current	# in meters
float32 theta_current	# in radians
bool ignore_rotation 	#if true robot is not rotated after moving. if false robot is rotated "rotation" relative to the starting orientation
